packages/gestalt/src/Image/Image.css

```diff
@@ -4,6 +4,15 @@
   composes: absolute from "../Layout.css";
 }
 
+/*
+    Disables alt text showing when loading an image in firefox
+    https://stackoverflow.com/questions/12667926
+*/
+
+img[alt] {
```

Comment:
This will be global for all images - I’m pretty sure that’s not you want here. Can we make this a class?